CHANGELOG.md

```diff
@@ -12,14 +12,15 @@
 * [CHANGE] Discovery: Removed `prometheus_sd_kubernetes_cache_*` metrics. Additionally `prometheus_sd_kubernetes_workqueue_latency_seconds` and `prometheus_sd_kubernetes_workqueue_work_duration_seconds` metrics now show correct values in seconds. #6393
 * [CHANGE] Remote write: Changed `query` label on `prometheus_remote_storage_*` metrics to `remote_name` and `url`. #6043
 * [FEATURE] API: Added new endpoint for exposing per metric metadata `/metadata`. #6420 #6442
+* [FEATURE] API: Added new endpoint for exposing tsdb status `/status/tsdb`. #6281
```

Comment:
Don't mention it twice, just the enhancement here I guess.

Reply:
Fixed.